dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/response/SolrSearchResult.java

```diff
@@ -614,6 +626,11 @@ public String getFileDatasetUrl() {
                 : "/dataset.xhtml?id=" + parent.getId() + "&versionId=" + datasetVersionId;
     }
 
+    public boolean isUnderEmbargo() {
+        return embargoUntil != null && Instant.now().atZone(ZoneId.systemDefault())
+                .isBefore(embargoUntil.toInstant().atZone(ZoneId.systemDefault()));
```

Comment:
Is there a reason why we convert two dates from `Instant` to `ZonedDateTime` before comparing them? Isn't comparing two `Instant` would have the same effect?

Reply:
Not sure why I added it, to be honest. It should be exactly the same. Removed the conversion.